docs/en/UI/AspNetCore/Tag-Helpers/Collapse.md

```diff
@@ -0,0 +1,92 @@
+# Collapse
+
+## Introduction
+
+`abp-collapse-body` is the main container for showing and hiding content. `abp-collapse-id` is used to show and hide the content container. Can be triggered with both `abp-button` and `a` tags.
+
+Basic usage:
+
+````xml
+<abp-button button-type="Primary" abp-collapse-id="collapseExample" text="Button with data-target" />
+<a abp-button="Primary" abp-collapse-id="collapseExample"> Link with href </a>
+
+<abp-collapse-body id="collapseExample">       
+                    Anim pariatur wolf moon tempor,,, sunt aliqua put a bird on it squid single-origin coffee nulla assumenda shoreditch et. Nihil anim keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident. Ad vegan excepteur butcher vice lomo. Leggings occaecat craft beer farm-to-table, raw denim aesthetic synth nesciunt you probably haven't heard of them accusamus labore sustainable VHS.
+</abp-collapse-body>
+````
+
+
+
+## Demo
+
+See the [collapse demo page](https://bootstrap-taghelpers.abp.io/Components/Collapse) to see it in action.
+
+## Attributes
+
+### show
+
+A value indicates if the collapse body will be initialized visible or hidden. Should be one of the following values:
+
+* `false` (default value)
+* `true`
+
+### multi
+
+A value indicates if an `abp-button` or `a` can show and hide multiple collapse body element. Should be one of the following values:
```

Comment:
This is not right. We use this attribute only on `abp-collapse-body` tag. 
We don't need "multi" attribute on `abp-button` or `a` because we can determine it automatically. But we need it on `abp-collapse-body` tag. See example below.

So this should be: 
"A value indicates if an `abp-collapse-body` can be shown or hidden by an element that can show/hide multiple collapse bodies. Basically, this attribute adds "multi-collapse" class to `abp-collapse-body`. Should be one of the following values:"

Reply:
@gterdem i approved changes by mistake.

Reply:
Corrected.